### PR TITLE
Add cilium agent workflow

### PR DESCRIPTION
**Pull request by chez-shanpu**

### CODE REVIEW

File: cilium/Dockerfile [hunk 1]
```diff
@@ -37,8 +37,8 @@ RUN mkdir -p ${DESTDIR} \
 
 
 FROM build-base as builder
-COPY workspace/bin/llvm-objcopy /bin/
-COPY workspace/bin/clang workspace/bin/llc /bin/
+COPY ./src/workspace/bin/llvm-objcopy /bin/
+COPY ./src/workspace/bin/clang src/workspace/bin/llc /bin/
```
[the remark this answers is not in the document]
Comment (chez-shanpu) [reply cilium/Dockerfile]: fixed it

**Review by terassyi — DISMISSED** [review]
LGTM

**Review by yokaze — CHANGES_REQUESTED** [review]

File: cilium/README.md [hunk 2]
```diff
@@ -6,4 +6,4 @@ This directory provides the Dockerfile for building a container image containing
 Docker images
 -------------
 
-Docker images are available on [Quay.io](https://quay.io/repository/cybozu/cilium)
+Docker images are available on [ghcrio](https://github.com/cybozu/neco-containers/pkgs/container/cilium)
```
[the remark this answers is not in the document]
Comment (chez-shanpu) [reply cilium/README.md]: fixed it
thank you :)

File: .github/actions/build_cilium-agent/action.yaml [hunk 3]
```diff
@@ -0,0 +1,68 @@
+name: "Build Cilium Agent"
+description: "Building cilium-agent"
+inputs:
+  request-scan:
+    description: "Request image scanning"
+    required: false
+    default: "false"
```
[the remark this answers is not in the document]
Comment (chez-shanpu) [reply .github/actions/build_cilium-agent/action.yaml]: I removed `request-scan` and set "true".

File: cilium/Dockerfile [hunk 4]
```diff
@@ -56,15 +56,16 @@ RUN apt-get install -y --no-install-recommends binutils \
       libelf-dev \
       make \
       unzip \
+      libz3-dev \
     && images/builder/install-protoc.sh \
     && make build-container install-container-binary
 
 FROM ${UBUNTU_IMAGE}
 ARG DESTDIR
```
[the remark this answers is not in the document]
Comment (chez-shanpu) [reply cilium/Dockerfile:66]: I added it.

File: .github/actions/build_cilium-agent/action.yaml [hunk 5]
```diff
@@ -0,0 +1,68 @@
...
+        github_token: ${{ inputs.github_token }}
+
+    - id: extract
+      name: Extract targets
+      uses: ./.github/actions/extract_tags
+      with:
+        dir: ./cilium
+        container-image: cilium
+        github_token: ${{ inputs.github_token }}
+        request-scan: ${{ inputs.request-scan }}
```
[the remark this answers is not in the document]
Comment (chez-shanpu) [reply .github/actions/build_cilium-agent/action.yaml]: I fixed it.

**Review by yokaze — APPROVED** [review]
LGTM!

**Pull request merged**